pydis_site/apps/api/viewsets/bot/off_topic_channel_name.py

```diff
@@ -106,6 +110,31 @@ def list(self, request: HttpRequest) -> Response:
                     'random_items': ["Must be a positive integer."]
                 })
 
+            if 'mark_used' in request.query_params and request.query_params['mark_used']:
+                queryset = self.get_queryset().order_by('?').exclude(used=True)[:random_count]
```

Comment:
what does `.order_by('?')` do here?

Reply:
This makes that ORM will return queries in random order. Removing this will make that we get always the same names.

Reply:
ah, nice.